internal/inspectimage/bom_display.go

```diff
@@ -48,3 +70,31 @@ func displayBOM(bom []buildpack.BOMEntry) []BOMEntryDisplay {
 
 	return result
 }
+
+func displayBOMWithExtension(bom []buildpack.BOMEntry) []BOMEntryWithExtensionDisplay {
```

Comment:
Based on my previous comment, if the struct name is changed to `BOMEntryWithExtension` then the method's name make sense to what is returning.